cmd/kritis/kubectl/plugins/resolve/README.md

```diff
@@ -3,46 +3,57 @@
 resolve-tags replaces tagged images in your Kubernetes yamls with their corresponding digests, and prints the new manifest to STDOUT.
 It can be run as a binary or installed as a kubectl plugin.
 
-## kubectl plugin
+## Installation
 
-To install as a kubectl plugin, run:
+## macOS
 
+```shell
+curl -LO https://storage.googleapis.com/resolve-tags/latest/resolve-tags-darwin-amd64.tar && \
+RESOLVE_TAGS_DIR=$HOME/.kube/plugins/resolve && \
+mkdir -p $RESOLVE_TAGS_DIR && tar -C $RESOLVE_TAGS_DIR -xf resolve-tags-darwin-amd64.tar && \
+mv $RESOLVE_TAGS_DIR/resolve-tags-darwin-amd64 $RESOLVE_TAGS_DIR/resolve-tags && \
+sudo cp $RESOLVE_TAGS_DIR/resolve-tags /usr/local/bin/
 ```
-make install-plugin
+
+## Linux
+
+```shell
+
+curl -LO https://storage.googleapis.com/resolve-tags/latest/resolve-tags-linux-amd64.tar curl  && \
+RESOLVE_TAGS_DIR=$HOME/.kube/plugins/resolve && \
+mkdir -p $RESOLVE_TAGS_DIR && tar -C $RESOLVE_TAGS_DIR -xf resolve-tags-linux-amd64.tar && \
+mv $RESOLVE_TAGS_DIR/resolve-tags-linux-amd64 $RESOLVE_TAGS_DIR/resolve-tags && \
```

Comment:
Is the mv necessary? You should be able to combine the rename and "sudo cp":

sudo cp ${RESOLVE_TAGS_DIR}/resolve-tags-linux-amd64 /usr/local/bin/resolve-tags

Reply:
So we want the binary in two places, in the `~/.kube/plugins/resolve` directory so it can run as a kubectl plugin, and in `/usr/local/bin` so it can be run independently